osu.Game/Beatmaps/BeatmapMetadata.cs

```diff
@@ -66,5 +66,23 @@ public string AuthorString
             Source,
             Tags
         }.Where(s => !string.IsNullOrEmpty(s)).ToArray();
+
+        public override bool Equals(object other)

```

Comment:
Don't override `object.Equals`. Use `IEqutable<T>`.

Reply:
Good catch. Looking for references how this was done before I totally missed this detail. Will change.

Reply:
I did a lookup for all `object.Equals` references, and the only remained instance was `CultureInfo`.